Jellyfin.Plugin.KodiSyncQueue/Utils/Helpers.cs

```diff
@@ -0,0 +1,75 @@
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Model.Entities;
+using MediaType = Jellyfin.Plugin.KodiSyncQueue.Entities.MediaType;
+
+namespace Jellyfin.Plugin.KodiSyncQueue.Utils
+{
+    public static class Helpers
+    {
+        public static bool FilterItem(BaseItem item, out MediaType type)
```

Comment:
I suggest renaming it to better show what it does... maybe something like `GetItemType` or `FilterAndGetItemType`?